frontend/packages/kubevirt-plugin/src/components/vms/vm-console.tsx

```diff
@@ -91,18 +111,30 @@ const VMConsoles: React.FC<VMConsolesProps> = ({
   ) : null;
 
   return (
-    <div className="co-m-pane__body">
-      <AccessConsoles preselectedType={VNC_CONSOLE_TYPE} disconnectByChange={false}>
-        <SerialConsoleConnector {...serial} />
-        <VncConsole {...vnc} />
-        {desktopViewverSelector}
+    <>
+      {showOpenInNewWindow && (
+        <div className="vm-console-actions">
+          <Button
+            component="a"
+            target="_blank"
+            variant="secondary"
+            href={`/k8s/ns/${namespace}/virtualmachineinstances/${vmName}/standaloneconsole?type=${type.toString()}`}
+          >
+            Open Console in new Window
+          </Button>
+        </div>
+      )}
+      <AccessConsoles preselectedType={type?.toPatternflyLabel()} disconnectByChange={false}>
+        {serial && <SerialConsoleConnector {...serial} />}
```

Comment:
this is not sufficient for checking the presence of serial or vnc. It just checks if VMI is running